scikeras/wrappers.py

```diff
@@ -1611,6 +1649,75 @@ class KerasRegressor(BaseWrapper):
         **BaseWrapper._tags,
     }
 
+    def __init__(
+        self,
+        model: Union[None, Callable[..., tf.keras.Model], tf.keras.Model] = None,
+        *,
+        build_fn: Union[
+            None, Callable[..., tf.keras.Model], tf.keras.Model
+        ] = None,  # for backwards compatibility
+        warm_start: bool = False,
+        random_state: Union[int, np.random.RandomState, None] = None,
+        optimizer: Union[
+            str, tf.keras.optimizers.Optimizer, Type[tf.keras.optimizers.Optimizer]
+        ] = "rmsprop",
+        loss: Union[
+            Union[str, tf.keras.losses.Loss, Type[tf.keras.losses.Loss], Callable], None
+        ] = "auto",
+        metrics: Union[
+            List[
+                Union[
+                    str,
+                    tf.keras.metrics.Metric,
+                    Type[tf.keras.metrics.Metric],
+                    Callable,
+                ]
+            ],
+            None,
+        ] = None,
+        batch_size: Union[int, None] = None,
+        validation_batch_size: Union[int, None] = None,
+        verbose: int = 1,
+        callbacks: Union[
+            List[Union[tf.keras.callbacks.Callback, Type[tf.keras.callbacks.Callback]]],
+            None,
+        ] = None,
+        validation_split: float = 0.0,
+        shuffle: bool = True,
+        run_eagerly: bool = False,
+        epochs: int = 1,
+        **kwargs,
+    ):
+
+        # Parse hardcoded params
+        self.model = model
+        self.build_fn = build_fn
+        self.warm_start = warm_start
+        self.random_state = random_state
+        self.optimizer = optimizer
+        self.loss = loss
+        self.metrics = metrics
+        self.batch_size = batch_size
+        self.validation_batch_size = validation_batch_size
+        self.verbose = verbose
+        self.callbacks = callbacks
+        self.validation_split = validation_split
+        self.shuffle = shuffle
+        self.run_eagerly = run_eagerly
+        self.epochs = epochs
+
+        # Unpack kwargs
+        vars(self).update(**kwargs)
+
+        # Save names of kwargs into set
+        if kwargs:
+            self._user_params = set(kwargs)
```

Comment:
This is literally just to add a default value for loss.

Reply:
Why is `self._user_params` required? I'm not seeing any usage.

Reply:
It is used by BaseWrapper. But I now realize I made a bonehead mistake and should have just called `super().__init__(...)`, which will in turn do this `self._user_params` logic. Not sure if that is what you were referring to or if you were referring to the existence of `self._user_params` altogether. If it's the latter, I think it's beyond the scope of this PR.

Reply:
Replaced with a `super().__init__(...)` call.